DXRModules/DeusEx/Classes/DXRKeys.uc

```diff
@@ -502,6 +518,12 @@ function ApplyDoorFixes()
                 d.minDamageThreshold = door_fixes[i].minDamageThreshold;
             if(door_fixes[i].doorStrength > 0)
                 d.doorStrength = door_fixes[i].doorStrength;
+
+            if(door_fixes[i].bHighlight==false){
+                d.bFrobbable = false;
+                d.bBreakable = false;
```

Comment:
I don't think we should be setting breakable to false here? maybe you meant to do bPickable?

Reply:
I probably should have actually called this like... "non-interactible" or something